### PR TITLE
Updates /upgrade/sharding pre-merge copy

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Updates path for `/shard-chains/` to `/sharding/`
- Updates file names of all intl markdown paths
- Updates all links to new path
- Adds redirect
- Updates portions of the Sharding page to remove usage of "shard chains" and adds an InfoBanner warning that the content below a certain portion of the page is outdated.
- Links out to Hitchhikers Guide to Ethereum (💪 )

Acts as a temporary communication measure to let people know this info is no longer up-to-date, and limits scope creep of revamping this entire page as part of the Merge prep epic. 

## Related Issue
- #6281
- #6590

### CODE REVIEW

File: src/content/contributing/translation-program/content-buckets/index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Below is a breakdown of the website pages each content bucket contains.
 - [Ethereum vision](/upgrades/vision/)
 - [The Beacon Chain](/upgrades/beacon-chain/)
 - [The Merge](/upgrades/merge/)
-- [Shard chains](/upgrades/shard-chains/)
+- [Shard chains](/upgrades/sharding/)
```
Comment (corwintines) [inline src/content/contributing/translation-program/content-buckets/index.md:52]:
```suggestion
- [Sharding](/upgrades/sharding/)
```

File: src/content/developers/docs/scaling/plasma/index.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Conversely, plasma chains derive their security from Mainnet. This makes them me
 
 ### Plasma vs sharding {#plasma-vs-sharding}
 
-Both plasma chains and [shard chains](/upgrades/shard-chains/) periodically publish cryptographic proofs to Ethereum Mainnet. However, both have different security properties.
+Both plasma chains and [shard chains](/upgrades/sharding/) periodically publish cryptographic proofs to Ethereum Mainnet. However, both have different security properties.
```
Comment (corwintines) [inline src/content/developers/docs/scaling/plasma/index.md:39]:
```suggestion
Both plasma chains and [sharding](/upgrades/sharding/) periodically publish cryptographic proofs to Ethereum Mainnet. However, both have different security properties.
```

File: src/content/upgrades/sharding/index.md [hunk 3]
```diff
@@ -35,18 +35,23 @@ With lower hardware requirements, sharding will make it easier to run [clients](
...
-  At first, you'll need to run a Mainnet client at the same time as your Beacon Chain client. <a href="https://launchpad.ethereum.org" target="_blank">The launchpad</a> will walk you through the hardware requirements and process. Alternatively you can use a <a href="/developers/docs/apis/backend/#available-libraries">backend API</a>.
+<InfoBanner isWarning>
+  At first, you'll need to run a Mainnet (execution layer) client at the same time as your Beacon Chain (consensus later) client. <a href="https://launchpad.ethereum.org" target="_blank">The launchpad</a> will walk you through the hardware requirements and process.
 </InfoBanner>
 
 ## Shard chains version 1: data availability {#data-availability}
 
+<InfoBanner emoji=":construction:" isWarning>
+  <strong>Note:</strong> The plans for sharding have been evolving as more efficient paths to scaling have been developed. "Danksharding" is a new approach to sharding, which does not utilize the concept of shard "chains" but instead uses shard "blobs" to split up the data, along with "data availability sampling" to confirm all data has been made available. This change in plan solves the same original problem.<br/><br/>
```
Comment (corwintines) [inline src/content/upgrades/sharding/index.md:45]:
```suggestion
  <strong>Note:</strong> The plans for sharding have been evolving as more efficient paths to scaling have been developed. "Danksharding" is a new approach to sharding, which does not utilize the concept of shard "chains" but instead uses shard "blobs" to split up the data, along with "data availability sampling" to confirm all data has been made available. This change in plan solves the same original problem.
```

Dont think these have any opening tags

**Review by corwintines — APPROVED** [review]

**Review by minimalsm — APPROVED** [review]
I can't review this properly because the files changed tab keeps crashing my browser 😅 changes in the upgrades JSON look ok to me. (Didn't notice any major updates, LMK if there is anything you'd like my eyes on that I've missed)

**Pull request merged**